packages/terra-framework-docs/src/terra-dev-site/doc/folder-tree/Examples.2/BasicFolderTree.1.doc.mdx

```diff
@@ -1,3 +1,38 @@
 import BasicFolderTree from './BasicFolderTree?dev-site-example';
 
+# Basic Folder Tree
+
+### Description
+
+The following example demonstrates a basic folder tree. For more details on specific folder tree
+features, see the following examples:
+* [Selection](./folder-tree-selection)
+* [Expand/Collapse](./folder-tree-expand-collapse)
+* [Expand/Collapse All](./folder-tree-expand-collapse-all)
+
+A folder tree is composed of two components:
+* **FolderTree**: The top-level folder tree component.
+* **FolderTreeItem**: A row in a folder tree that may have `subfolderItems`.
+  * If there are 1 or more `subfolderItems`, the row will be expandable/collapsible and display a folder icon.
+
+All children of a FolderTree **must** be FolderTreeItems.
+
+### Properties required
+
+The following FolderTree props are required for a basic folder tree:
+* **title**: The title of the folder tree. This is also used as the `aria-label` for the folder tree.
```

Comment:
Just an observation, in similar case [here](https://engineering.cerner.com/terra-ui/components/cerner-terra-framework-docs/compact-interactive-list/examples/cell-content#required-properties) the technical writer made the props names to be links, we might want to follow that for consistency.

Reply:
Updated: [0f5b7db](https://github.com/cerner/terra-framework/pull/1973/commits/0f5b7db7c0699ee76faaecbf1800596272a2bb11)